fields.yml

```diff
@@ -982,19 +980,28 @@
             Sometimes called program name or similar.
           example: ssh
     
-        - name: pid
-          level: core
-          type: long
-          description: >
-            Process id.
-          exmple: ssh
-    
         - name: ppid
           level: extended
           type: long
           description: >
             Process parent id.
     
+        - name: args
+          level: extended
+          type: keyword
+          description: >
+            Process arguments.
+    
+            May be filtered to protect sensitive information.
+          example: ["-l", "user", "10.0.0.16"]
```

Comment:
This example was wrong because `args` should include all the arguments. It's missing `args[0]` which is the thing that was executed.

Reply:
Good catch